### PR TITLE
Manage data transforms in Redpanda Console

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2386
Review deadline: 14 July

## Page previews

https://deploy-preview-595--redpanda-docs-preview.netlify.app/24.2/manage/console/data-transforms/

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Review by rikimaru0345 — APPROVED** [review]

File: modules/manage/pages/console/data-transforms.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - Any lag (the amount of pending records on the input topic that have yet to be processed by the transform)
+
+[[logs]]
+== View logs
+
+To view logs for a transform function in Redpanda Console:
+
+. Navigate to the *Transforms* menu.
+. Click on the name of a transform function
+. Click the *Logs* tab to see the logs.
```
Comment (Feediver1) [inline modules/manage/pages/console/data-transforms.adoc]:
```suggestion
. Navigate to the *Transforms* menu.
. Click on the name of a transform function.
. Click the *Logs* tab to see the logs.
```

Comment (Feediver1) [reply modules/manage/pages/console/data-transforms.adoc]: Within these sections, sometimes there is end punctuation, sometimes not. Whichever you decide to go with, be consistent.

**Review by Feediver1 — APPROVED** [review]
please address punctuation consistency

**Review by vladoschreiner — APPROVED** [review]
LGTM.

Optional suggestion: connectors also use the message browser embedded in the connector window (as do transform functions, right?). Due to tech implications, this window goes only 3hrs back to the history, despite longer retention of the log topic. I saw users confused by lack of historical logging (e.g. connector failed overnight, they only found out in the morning). I suggest documenting that they can access the log topic using usual means (`rpk`, console message browser on the logs topic) to historical log analysis.

**Comment by JakeSCahill** [comment]
Thanks @vladoschreiner - the UI has a message that says: The logs below are for the last five hours.

To avoid hardcoding this time limit (in case it changes), I kept the doc more general: https://github.com/redpanda-data/docs/pull/595/commits/6712acd643974491812fe0c32cd43730476cdf25

**Pull request merged**